redfish.conf

```diff
@@ -15,6 +15,10 @@ api_version = 300
 userName = administrator
 password =
 
+[oneview_scmb]
+port = 5671
+socketTimeout = 5
```

Comment:
Why add the SCMB config to the conf file? Most likely these will never need to be changed and by allowing the user to change them it could break the connection to the SCMB.

Reply:
I don't think we want to let the user change it.  If they do we won't be able to connect to the OneView SCMB.  And I don't think HPE will ever change that port number.